packages/playwright-msw/package.json

```diff
@@ -27,6 +27,9 @@
     "@playwright/test": ">=1.20.0",
     "msw": ">=0.44.0"
   },
+  "dependencies": {
+    "@mswjs/cookies": "^0.2.2"
+  },
```

Comment:
Not sure whether it should be a peer dependency or not. I added it as a dependency to not require users to include it in their `package.json`.

Reply:
I agree with your reasoning, makes sense for it to be a normal dependency imo.